sql/mysqld.h

```diff
@@ -176,7 +176,13 @@ extern const char *log_output_str;
 extern const char *log_backup_output_str;
 extern char *temporal_current_timestamp;
 extern my_bool vers_force;
-extern my_bool vers_hide;
+enum vers_hide_enum: ulong {
```

Comment:
It do not work with `enum_binlog_format` and actually uses a different `ulong` [variable](https://github.com/tempesta-tech/mariadb_10.2/blob/427f834841f48cb0185a23536a6a59e8fb7cd221/sql/sql_class.h#L698).

This is even better.
```diff
diff --git a/sql/mysqld.cc b/sql/mysqld.cc
index e48988a..ce68980 100644
--- a/sql/mysqld.cc
+++ b/sql/mysqld.cc
@@ -786,7 +786,7 @@ char *opt_logname, *opt_slow_logname, *opt_bin_logname;
 /* System Versioning */
 char *temporal_current_timestamp;
 my_bool vers_force= false;
-vers_hide_enum vers_hide= VERS_HIDE_AUTO;
+ulong vers_hide= VERS_HIDE_AUTO;
 
 /* Static variables */
 
diff --git a/sql/mysqld.h b/sql/mysqld.h
index e06b9fa..2230ee2 100644
--- a/sql/mysqld.h
+++ b/sql/mysqld.h
@@ -176,13 +176,13 @@ extern const char *log_output_str;
 extern const char *log_backup_output_str;
 extern char *temporal_current_timestamp;
 extern my_bool vers_force;
-enum vers_hide_enum: ulong {
+enum vers_hide_enum {
   VERS_HIDE_AUTO= 0,
   VERS_HIDE_IMPLICIT,
   VERS_HIDE_FULL,
   VERS_HIDE_NEVER
 };
-extern vers_hide_enum vers_hide;
+extern ulong vers_hide;
 extern char *mysql_home_ptr, *pidfile_name_ptr;
 extern MYSQL_PLUGIN_IMPORT char glob_hostname[FN_REFLEN];
 extern char mysql_home[FN_REFLEN];
```